[C++] Add Protocol and websocket documentation (#174)
kuzzleio/documentation-V2: src/sdk-reference/cpp/1/websocket/listener-count/index.md

---
layout: sdk.html.hbs
title: listenerCount

> description: Count the number of listener you have on an event

Reviewer: ```suggestion
description: Returns the number of listeners attached on an event
```

---

# listenerCount

> Count the number of listener you have on an event

Reviewer: ```suggestion
Returns the number of listeners attached on an event.
```

## Signature

```cpp
int listenerCount(kuzzleio::Event event);
```

## Arguments

| Argument   | Type                      | Description
| ---------- | ------------------------- | ------------------------------------------------------------------------------------------------------

> | `event`    | <pre>kuzzleio::Event</pre>           | An enum representing the listener [event]({{ site_base_path }}sdk-reference/essentials/event-handling)

Reviewer: broken link

### **event**

One of the following event:

```cpp
CONNECTED
DISCARDED
DISCONNECTED
LOGIN_ATTEMPT
NETWORK_ERROR
OFFLINE_QUEUE_POP
OFFLINE_QUEUE_PUSH
QUERY_ERROR
RECONNECTED
JWT_EXPIRED
ERROR
```

## Return

> The number of listener you have on the specified event.

Reviewer: ```suggestion
The number of listeners attached on the specified event.
```
